glean-core/src/ping/mod.rs

```diff
@@ -48,7 +48,31 @@ impl PingMaker {
         Self
     }
 
-    fn get_ping_seq(&self, glean: &Glean, storage_name: &str) -> usize {
+    /// Set the next ping sequence number to the provided one.
+    ///
+    /// This function stores the next sequence number (the one that
+    /// will be returned next time `get_ping_seq` is called) in the
+    /// glean-core store. The main purpose of this function is to allow
+    /// overriding sequence numbers with migration data coming from
+    /// glean-ac. 
+    pub(super) fn set_ping_seq(&self, glean: &Glean, storage_name: &str, next_seq: i32) {
+        // Sequence numbers are stored as a counter under a name that includes the storage name
+        let seq = CounterMetric::new(CommonMetricData {
+            name: format!("{}#sequence", storage_name),
+            // We don't need a category, the name is already unique
+            category: "".into(),
+            send_in_pings: vec![INTERNAL_STORAGE.into()],
+            lifetime: Lifetime::User,
+            ..Default::default()
+        });
+
+        // It's safe to add `next_seq` because the glean-ac migration code
+        // clears the glean-core database before the migration starts.
+        seq.add(glean, next_seq);
+    }
+
+    /// This is pub(super) exclusively for enabling the migration tests.
```

Comment:
IMO the first line should still document the method.

I'd also change the string to "This is crate-internal exclusively for enabling the migration tests." or something along the lines.